prover/data/c12.stark_struct.json

```diff
@@ -1,10 +1,10 @@
 {
-  "nBits": 15,
```

Comment:
for evm, the nBits should be 20, and nBitsExt should be 21,  and the first element of steps should be 21 too.